packages/sdk/src/modules/did/index.ts

```diff
@@ -34,16 +34,42 @@ interface CreateDidOptions {
   customDocumentFields?: CustomDocumentFields;
 }
 
+interface ReadDidOptions {
+  name: string;
+  address?: Address;
+}
+
+interface UpdateDidOptions {
```

Comment:
New objects being passed into the function. Should any members of the object be added/removed?